README.md

```diff
@@ -39,6 +41,8 @@ Quick sample of Azure Service Bus SDK
 Explain the layering
 Is this necessary?
 
+> Why do you need to talk about the Azure SDK?
```

Comment:
At first I was thinking it might be helpful for them to understand the code pieces but I figured it is actually a waste of time. The samples should be simply enough. I'll remove that part